src/errors.rs

```diff
@@ -0,0 +1,24 @@
+error_chain!{
```

Comment:
Non-actionable, emotional opinion based reaction: "ugh, error_chain magic contamination"

Reply:
git-rsl uses it, so I used it here.